support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -695,8 +696,8 @@ function CheckoutComponent({ geoId }: Props) {
 						>
 							<Box cssOverrides={shorterBoxMargin}>
 								<BoxContents>
-									<div css={personalDetailsFieldGroupStyles(true)}>
-										<h2 css={personalDetailsHeader}>Your details</h2>
+									<fieldset css={fieldset}>
```

Comment:
As this css is related to a single fieldset and not all, would its be worth giving it a longer name like `fieldsetPersonalDetails`?

Reply:
I've added this to payment fields too - but I am not sure we should as the CSS is using first-of-type, so I might try a refactor post-merge (so we can start demoing this).